raas_v2/api_helper.py

```diff
@@ -310,7 +310,8 @@ class RFC3339DateTime(CustomDate):
 
         @classmethod
         def from_datetime(cls, date_time):

```

Comment:
While using [a method documented as having _optional_ inputs](https://github.com/tangocard/raas-v2-sdk-python/blob/91cd237b7921756eb6178d0b7e3703f4b943c7d7/raas_v2/controllers/orders_controller.py#L197) I hit errors if I did not provided a datetime for the "optional" startDate (see traceback below).

Am wondering though if it's safe to make this a global change, or if I should subclass the RFC3339DateTime object to make an OptionalRFC3339DateTime with this conditional?

```
INFO:raas_v2.controllers.orders_controller:get_orders called.
INFO:raas_v2.controllers.orders_controller:Preparing query URL for get_orders.
ERROR:raas_v2.controllers.orders_controller:'NoneType' object has no attribute 'isoformat'
Traceback (most recent call last):
  File "/Users/kborg/dev/__misc/temp_tango/env/lib/python2.7/site-packages/raas_v2/controllers/orders_controller.py", line 239, in get_orders
    'startDate': APIHelper.RFC3339DateTime(options.get('start_date', None)),
  File "/Users/kborg/dev/__misc/temp_tango/env/lib/python2.7/site-packages/raas_v2/api_helper.py", line 264, in __init__
    self.value = self.from_datetime(dtime)
  File "/Users/kborg/dev/__misc/temp_tango/env/lib/python2.7/site-packages/raas_v2/api_helper.py", line 313, in from_datetime
    return date_time.isoformat()
AttributeError: 'NoneType' object has no attribute 'isoformat'
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-31-fe516414b137> in <module>()
----> 1 orders_client.get_orders({'account_identifier': 'fold-dev-inventory'})

/Users/kborg/dev/__misc/temp_tango/env/lib/python2.7/site-packages/raas_v2/controllers/orders_controller.pyc in get_orders(self, options)
    237                 'customerIdentifier': options.get('customer_identifier', None),
    238                 'externalRefID': options.get('external_ref_id', None),
--> 239                 'startDate': APIHelper.RFC3339DateTime(options.get('start_date', None)),
    240                 'endDate': APIHelper.RFC3339DateTime(options.get('end_date', None)),
    241                 'elementsPerBlock': options.get('elements_per_block', None),

/Users/kborg/dev/__misc/temp_tango/env/lib/python2.7/site-packages/raas_v2/api_helper.pyc in __init__(self, dtime, value)
    262             self.datetime = dtime
    263             if not value:
--> 264                 self.value = self.from_datetime(dtime)
    265             else:
    266                 self.value = value

/Users/kborg/dev/__misc/temp_tango/env/lib/python2.7/site-packages/raas_v2/api_helper.pyc in from_datetime(cls, date_time)
    311         @classmethod
    312         def from_datetime(cls, date_time):
--> 313             return date_time.isoformat()
    314 
    315         @classmethod

AttributeError: 'NoneType' object has no attribute 'isoformat'
```

Reply:
> I would make the change in the invocation
yes! that makes sense, will do that. thanks.